pkg/daemon/update.go

```diff
@@ -400,6 +403,11 @@ func getFileOwnership(file ignv2_2types.File) (int, int, error) {
 	return uid, gid, nil
 }
 
+func (dn *Daemon) updateOS(oldConfig, newConfig *mcfgv1.MachineConfig) error {
```

Comment:
even though this isn't exported, it's probably worth adding a comment describing it's functionality to keep it in line with the other functions that perform parts of an update.